src/form/actionCreators.js

```diff
@@ -31,9 +30,9 @@ export const clear = createAction(
 
 export const initialise = createAction(
   INITIALISE,
-  payload => omit(payload, ['form']),
+  ({ form, ...payload }) => payload,
   payload => ({
```

Comment:
Don't even need to initialize a variable when we pick out keys. How does this look?
```js
export const initialise = createAction(
  INITIALISE,
  ({ form, ...payload }) => payload,
  ({ form }) => form,
)
```